### DISCUSSION TITLE
SearchAll :: What makes a torrent "suitable?"

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [cross-seed/cross-seed#255] [Q&A] [answered]
I helped a friend of mine get this up and running on his system as well. But I noticed he had a couple of TV show torrents that were "not suitable" for scanning for some reason. I wasn't exactly sure what the SearchAll option did in the config, so I turned it on and scanned again. This time it read the files, as I expected it would. My question is, what exactly makes a torrent "suitable?"

I also noticed cross-seed didn't seem to work on music torrents, but I haven't tried with SearchAll set to true. I was hoping for a little insight on what to look for in terms of being suitable for cross-seed to be able to search it.

**Comment by mmgoodnow** [comment, marked as answer]
https://github.com/mmgoodnow/cross-seed/discussions/91#discussioncomment-895926 has some info on this
also https://github.com/mmgoodnow/cross-seed/discussions/232#discussioncomment-2666509

The fact that the defaults are what they are is mostly a vestige of cross-seed's early days.

**Reply by SawkeeReemo** [reply]
This explains everything for me, thank you! I’m a big music seeder, so I might do a one-time searchAll cross-seed check per tracker on those, and then move on. I appreciate the info and your work here. I love all things automation.
